app/src/main/res/drawable/bedtime.xml

```diff
@@ -5,5 +5,5 @@
     android:viewportHeight="960">
     <path
```

Comment:
Really weird that the icon itself wasn't aligned.
Just downloaded the vector asset again and it was centered